### PR TITLE
Change Item class simple getter-like functions into property getters.

### LINKED ISSUE #91: Replace getter function used in twee scripts with proper getters?
Just to make texts cleaner a bit we could replace `Name()` and similar functions with `get Name()` (and etc.) to save a couple of types and make it read simpler. Opening and issue to discuss the set of functions.

As to me, all argument-less functions in the `Item` class and its successors except of `Charges()` and those acessing `this.Inventory` object can be replaced.

### PR DESCRIPTION
**Pull request by ezsh**
This should shorten a bit references to them from twee code. Closes #91.

### CODE REVIEW

File: src/000-SCRIPT_OBJ/Items.js [hunk 1]
```diff
@@ -601,55 +609,47 @@ App.Items.Clothing = class Clothing extends App.Item {
...
      */
-    Color() {
-        return this.Data["Color"];
-    }
-
-    /**
-     * Type of clothing piece.
-     * @returns {string}
-     */
-    Type() {
```
Comment (JJM152) [inline src/000-SCRIPT_OBJ/Items.js]: Is this not used or something?

Comment (JJM152) [reply src/000-SCRIPT_OBJ/Items.js]: Nm, I see what's going on.

**Comment by ezsh** [comment]
With respect to #132 this can be extended to include the NPC class as well.

**Comment by JJM152** [comment]
Resolved the conflicts and testing. May revert if necessary.

**Comment by JJM152** [comment]
Seems fine. I need to get used to this new IDE I installed.

**Comment by ezsh** [comment]
Thanks!

**Pull request merged**